app/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,144 @@
+    <string name="onboarding_background_location">يحتاج Trail Sense إلى إذن لاستخدام GPS الخاص بك</string>
+    <string name="onboarding_location_weather">يسمح بتنبؤات الطقس أكثر دقة (يضبط الارتفاع)</string>
+    <string name="onboarding_location_navigation">يسمح لك بالانتقال إلى منارات</string>
+    <string name="onboarding_location_astronomy">يتيح لك رؤية شروق الشمس والقمر وتعيين أوقات</string>
+    <string name="onboarding_location_privacy">تحترم Trail Sense خصوصيتك - لا تترك بيانات الموقع جهازك أبدًا</string>
+    <string name="onboarding_location_all_time">يرجى تحديد \"السماح طوال الوقت \" من مربع حوار الإذن</string>
+    <string name="continue_button">استمر</string>
+    <string name="pref_onboarding_completed" translatable="false">onboarding_completed</string>
+    <string name="moon_rise">طلوع القمر</string>
+    <string name="moon_set">القمر</string>
+    <string name="stop_monitoring">قف</string>
+    <string name="pref_weather_category" translatable="false">pref_weather_category</string>
+    <string name="pref_compass_filter_amt" translatable="false">pref_compass_filter_amt</string>
+    <string name="pref_compass_filter_amt_title">تنعيم البوصلة</string>
+    <string name="pref_compass_filter_amt_summary">قد تؤدي القيم العالية إلى تقليل الضوضاء</string>
+    <string name="pref_theme" translatable="false">pref_theme</string>
+    <string name="pref_theme_title">موضوع</string>
+    <string name="pref_use_experimental_compass" translatable="false">pref_use_experimental_compass</string>
+    <string name="pref_use_experimental_compass_title">استخدم بوصلة تجريبية</string>
+    <string name="pref_use_legacy_compass_title">استخدم بوصلة قديمة</string>
+    <string name="pref_use_experimental_compass_summary">قد إصلاح مشاكل البوصلة</string>
+    <string name="pref_display_multi_beacons" translatable="false">pref_display_multi_beacons</string>
+    <string name="pref_display_multi_beacons_title">عرض منارات قريبة</string>
+    <string name="pref_storm_alert_sensitivity" translatable="false">pref_storm_alert_sensitivity</string>
+    <string name="pref_storm_alert_sensitivity_title">حساسية تنبيه العاصفة</string>
+    <string name="pref_show_ruler" translatable="false">pref_show_ruler</string>
+    <string name="pref_show_ruler_title">إظهار المسطرة</string>
+    <string name="pref_show_sun_moon_compass" translatable="false">pref_show_sun_moon_compass</string>
+    <string name="pref_show_sun_moon_compass_title">عرض الشمس /القمر على البوصلة</string>
+    <string name="pref_show_linear_compass" translatable="false">pref_show_linear_compass</string>
+    <string name="pref_show_linear_compass_title">إظهار البوصلة الخطية</string>
+    <string name="pref_show_linear_compass_summary">عندما يكون الهاتف عموديًا</string>
+
+    <string name="delete_beacon_confirmation">هل أنت متأكد أنك تريد إزالة "٪s"؟</string>
+
+    <plurals name="last_hours">
+        <item quantity="one">الاخير %d ساعة</item>
```

Comment:
This needs to have the following quantities defined here: few, many, two, zero

Reply:
there is all the quantities defined as defined in the main strings.xml